{{cookiecutter.repo_name}}/setup.py

```diff
@@ -40,20 +44,33 @@
     # Customize MANIFEST.in if the general case does not suit your needs
     # Comment out this line to prevent the files from being packaged with your software
     include_package_data=True,
-
+    python_requires=">=3.8",          # Python version restrictions
     # Allows `setup.py test` to work correctly with pytest
     setup_requires=[] + pytest_runner,
-
+    # Required packages, pulls from pip if needed
+    # do not use for Conda deployment
+    install_requires=[
+        "mdanalysis>=2.0.0",
+    ],
     # Additional entries you may want simply uncomment the lines you want and fill in the data
-    # url='http://www.my_package.com',  # Website
-    # install_requires=[],              # Required packages, pulls from pip if needed; do not use for Conda deployment
+    # url='{{cookiecutter.repo_name}}.readthedocs.io/en/latest/',  # Website
     # platforms=['Linux',
     #            'Mac OS-X',
     #            'Unix',
     #            'Windows'],            # Valid platforms your code works on, adjust to your flavor
-    # python_requires=">=3.5",          # Python version restrictions
 
     # Manual control if final package is compressible or not, set False to prevent the .egg from being made
     # zip_safe=False,
 
+    extras_require={
+        "test": [
+            "pytest>=6.0",
+            "pytest-xdist>=2.5",
+            "pytest-cov>=3.0",
+        ],
+        "doc": [
+            "sphinx>=1.8",
```

Comment:
As above re: minimum sphinx version.

Reply:
Changed to not specify a version at all!